lib/collection.js

```diff
@@ -1075,6 +1092,17 @@ Collection.prototype.updateMany = function(filter, update, options, callback) {
     options.ignoreUndefined = this.s.options.ignoreUndefined;
   }
 
+  var keys = Object.keys(update);
+
+  // same errors as the server would give for update doc lacking atomic operators
+  if (keys.length == 0) {
```

Comment:
prefer strict equality (`===`) and inequality (`!==`), we might consider adding an eslint rule for this.